event_sync/README.md

```diff
@@ -72,6 +72,7 @@ When an occupant joins, `POST ${api_prefix}/events/occupant/joined` is called wi
 * room_jid
 * is_breakout
 * breakout_room_id (only if is_breakout is true)
+* active_occupants
```

Comment:
Can we clarify in the docs that this is the new tally after the join? 

Some might wonder if this is the value at the point the join was initiated or if it includes the new participant. Easier to preempt questions like this in docs.

Reply:
Done.